### DISCUSSION TITLE
Repo error when building from clone

### DISCUSSION THREAD

**Discussion by tuppers360** [markdoc/markdoc#66] [Bug report] [answered]
Hi when cloning the repo and deploying to Vercel via the auto button I get the following error:

Type error: Cannot find name 'Prism'.
--
10:59:31.571 |  
10:59:31.571 | 8 \|
10:59:31.572 | 9 \|   React.useEffect(() => {
10:59:31.572 | > 10 \|     if (ref.current) Prism.highlightElement(ref.current, false);
10:59:31.572 | \|                      ^
10:59:31.572 | 11 \|   }, [children]);
10:59:31.572 | 12 \|
10:59:31.573 | 13 \|   return (
10:59:31.594 | error Command failed with exit code 1.
10:59:31.594 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:59:31.616 | Error! Command "yarn run build" exited with 1
10:59:31.718 | Error: Command "vercel build" exited with 1

**Comment by mfix-stripe** [comment, marked as answer]
@tuppers360 are you referring to this repo? https://github.com/markdoc/markdoc-starter

**Reply by tuppers360** [reply]
my apologies yes I am, as i say I used the deploy button but it failed to build, I have cloned another copy locally on my machine and get the same error unfortunately. 

Sorry for not being specific

**Reply by mfix-stripe** [reply]
No apologies necessary! This error should be fixed here: https://github.com/markdoc/markdoc-starter/commit/dcb90a4fcaaaa0d6802172787d2f051d7002f9e7

**Reply by tuppers360** [reply]
Super thanks, all works, I think I had a typo when trying to fix using 'prism' and not 'Prism'.

Just of note I had to delete the yarn.lock file as this was preventing the build from completing :-)
